ui/src/containers/Search/index.tsx

```diff
@@ -18,11 +25,22 @@ const Search: React.FC = () => {
     }, 20);
   });
 
-  const { resources } = useMst();
+  const debounced = useDebouncedCallback(
+    (value: string) => {
+      resources.setSearch(value);
+    },
+    // delay in ms
+    500
+  );
 
-  const inputRef = React.useRef<HTMLInputElement>(null);
+  const onSearchChange = () => {
+    const currentInput = inputRef.current;
+    assert(currentInput);
```

Comment:
Can we just assert like this 
```suggestion
    assert(inputRed.current);
```